CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Changelog
```

Comment:
@jurgenbelien the files from #191 are already in `main`. Do you know why they are in here again?

Reply:
I'll take a look, perhaps a over-eager rebase